### ISSUE TITLE
Default favicon is never loaded

### ISSUE THREAD

**Issue by glacambre** [tridactyl/tridactyl#701]
To reproduce this issue, `:tabopen https://example.org` then `:buffer `. The favicon for example.org will be an empty square. If you look at the src attribute of the favicon, you'll see that it is set to "#zXssPreventedz". I think this happens because of CSP issues.

**Comment by antonva** [comment]
I cannot reproduce this with `csp untouched` nor `csp clobber` on latest beta.

Are you still experiencing this @glacambre ?

**Comment by glacambre** [comment]
Sort of. With `csp untouched` I see the following favicons:
![screenshot_2018-10-04-11 57 56](https://user-images.githubusercontent.com/11534587/46466924-d83e0880-c7cc-11e8-9c00-8b9d66568602.png)

When I opened this issue, this is what I expected:
![screenshot_2018-10-04-12 01 22](https://user-images.githubusercontent.com/11534587/46467111-3cf96300-c7cd-11e8-8f57-bc9f078059cd.png)

What's different with what happened when I opened this issue is that the example.org favicon url isn't replaced with "zXssPrevented" anymore so the problem might be on Firefox's side instead of ours.

However, I still see the "zXssPrevented" value for tridactyl's newtab buffers.
